scripts/entrypoint.py

```diff
@@ -28,21 +28,33 @@ def dump_scriptvars():
         "TESTDROID_RUN_ID",
         "HOME",
         "HOSTNAME",
-        "TASKCLUSTER_ACCESS_TOKEN",
-        "TASKCLUSTER_CLIENT_ID",
+        # these are passed to g-w via config file
+        # "TASKCLUSTER_ACCESS_TOKEN",
+        # "TASKCLUSTER_CLIENT_ID",
```

Comment:
shall we just go ahead and remove these comments?

Reply:
Yeah, will do.